packages/ketcher-react/src/script/ui/views/toolbars/ToolbarGroupItem/ToolbarGroupItem.tsx

```diff
@@ -33,7 +33,7 @@ interface ToolbarGroupItemProps extends ToolbarItem {
 
 interface ToolbarGroupItemCallProps {
   onAction: (action: UiActionAction) => void
-  onOpen: (menuName: string, isSelected: boolean) => void
+  onOpen?: (menuName: string, isSelected: boolean) => void
```

Comment:
why did we change the common interface and make it less strict?
it will affect all other toolbars that depend on this interface.